### ISSUE TITLE
Call state disappears from API instead of being marked inactive

### ISSUE THREAD

**Issue by andy-vdg** [mutedeck/integrations#1]
My integration broke in HomeAssistant due to the fact that the API response has changed. 
It's been a long time since I set this up so I don't remember well but everything in my integration points to the fact that the API response when there is no active call should be: 
```
{
    "call": "inactive",    
    "control": "system",
    "mute": "inactive",
    "record": "disabled",
    "share": "disabled",
    "status": 200,
    "teams_api": "disabled",
    "video": "disabled"
}
```
This is what my further integration automation expects. It's not been working for a very long while and today I looked into why. It turns out that now whenever there is no call the API just returns: 
```
{
    "control": "system",
    "mute": "inactive",
    "record": "disabled",
    "share": "disabled",
    "status": 200,
    "teams_api": "disabled",
    "video": "disabled"
}
```
Because the call attribute is missing in Home Assistant it just records "Unknown" for the state of the sensor making further automations fail. I can work around it but just wanted to check that this wasn't changed by mistake.

**Comment by andy-vdg** [comment]
I added this to the sensor.yaml file to make it work again: 
```
    mutedeck_status_call:
      value_template: >
        {% if state_attr('sensor.mutedeck', 'call') is not none %}
          {{ state_attr('sensor.mutedeck', 'call') }}
        {% else %}
          inactive
        {% endif %}
```

**Comment by smitmartijn** [comment]
@andy-vdg thanks for reporting that. I'm not entirely sure when that field went away, mostly because other plugins don't expect it and work off the `control` field. I've just released MuteDeck v2.8 that does always have the `call` field and sets it to `inactive` if there's no call.
